[documentation]: enhance docs (#134)
hyperledger/iroha-javascript: README.md

## Maintenance

> Make sure you have installed Node.js v16.17 or v18. As for a package manager, this project uses [PNPM](https://pnpm.io/).

Reviewer: not v17?

Author: Odd versions of Node.js are unstable, even - long-term supported.

In general, it is not very important to mention a lot of versions here - v16.16 is enough.

Reviewer: ok

> Before working with the repo, install packages:

Reviewer: ```suggestion
Before working with the repository, install the necessary packages:
```

```bash
pnpm install
```

### Scripts

Most tasks are defined via [Jake](https://github.com/jakejs/jake), a JavaScript build tool. To explore available tasks, run:

```bash
pnpm jake -t
```

However, some tasks are defined directly in `package.json` and could be run with `pnpm run`:

```bash
# Run all project tests
pnpm test

# Perform type-checking
pnpm type-check

# Check/fix lint errors
pnpm lint
pnpm lint --fix

# Fix formatting
pnpm format:fix

> ```

Reviewer: - Run all project tests:

  ```bash
  pnpm test
  ```

- Perform type-checking:

  ```bash
  pnpm type-check
  ```

- Check or fix lint errors:

  ```bash
  pnpm lint
  pnpm lint --fix
  ```

- Fix formatting:

  ```bash
  pnpm format:fix
  ```

> Each monorepo package might have its own scripts and tasks. Please refer to a specific package's README for details.

Reviewer: ```suggestion
Each monorepo package might have its own scripts and tasks. You can find the package details in a README for that package.
```
